src/internet_identity/src/assets.rs

```diff
@@ -259,6 +259,7 @@ fn collect_assets_from_dir(dir: &Dir) -> Vec<(String, Vec<u8>, ContentEncoding,
             "png" => (file_bytes, ContentEncoding::Identity, ContentType::PNG),
             "svg" => (file_bytes, ContentEncoding::Identity, ContentType::SVG),
             "webp" => (file_bytes, ContentEncoding::Identity, ContentType::WEBP),
+            "woff2" => (file_bytes, ContentEncoding::GZip, ContentType::WOFF2),
```

Comment:
Is `ContentEncoding::GZip` accurate for non-gzipped `woff2`?

Reply:
No it's not! copy pasta

Reply:
https://github.com/dfinity/internet-identity/pull/1859